packages/paste-core/components/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Flex} from '@twilio-paste/flex';
+import {useTabPrimitiveState, TabPrimitiveInitialState} from '@twilio-paste/tabs-primitive';
+import {TabsContext} from './TabsContext';
+
+export type TabsProps = TabPrimitiveInitialState;
+
+// Set orientation to horizontal because undefined enables all arrow key movement
+const Tabs: React.FC<TabsProps> = ({children, orientation = 'horizontal', ...initialState}) => {
+  const tab = useTabPrimitiveState({orientation, ...initialState});
+  const value = React.useMemo(() => tab, Object.values(tab));
+  const returnValue = <TabsContext.Provider value={value}>{children}</TabsContext.Provider>;
```

Comment:
I really like this approach. I'm going to suggest we refactor Menu, which just exposes the hook, to do the same 👍